templates/conf.d/nginx.conf.erb

```diff
@@ -40,11 +40,16 @@ events {
   <%- end -%>
   worker_connections <%= @worker_connections -%>;
   <%- if @multi_accept == 'on' -%>
-    multi_accept on;
+  multi_accept on;
   <%- end -%>
   <%- if @events_use -%>
-    use <%= @events_use %>;
+  use <%= @events_use %>;
   <%- end -%>
+  <% if @debug_connection -%>
```

Comment:
If we simplify the data type to only allow Arrays, (and not `undef`), the template can be simplified too.

Reply:
done :)